ci_framework/roles/dlrn_promote/tasks/get_hashes.yml

```diff
@@ -1,21 +1,23 @@
 ---
-- name: Copy delorean.repo to dlrn_promote workspace
-  ansible.builtin.fetch:
-    src: "{{ cifmw_dlrn_promote_dlrnrepo_path }}"
-    dest: "{{ cifmw_dlrn_promote_workspace }}/"
-    flat: true
+- name: Make sure delorean.repo exists
+  ansible.builtin.stat:
+    path: "{{ cifmw_dlrn_promote_dlrnrepo_path }}"
+  register: dlrn_file
 
 - name: Get baseurls from delorean.repo
-  ansible.builtin.shell:
-    cmd: >-
-      set -o pipefail;
-      cat {{ cifmw_dlrn_promote_workspace }}/delorean.repo
-      | grep  'baseurl=' | cut -d '=' -f 2
-  register: base_urls
+  ansible.builtin.slurp:
+    src: "{{ cifmw_dlrn_promote_dlrnrepo_path }}"
+  register: dlrn_data
+  when: dlrn_file.stat.exists
   changed_when: true
```

Comment:
A slurp that flags the run as changed? Rare

Reply:
Done